pkg/processor/result_processor.go

```diff
@@ -12,8 +15,53 @@ import (
 	"github.com/aquasecurity/trivy-plugin-aqua/pkg/scanner"
 	"github.com/aquasecurity/trivy/pkg/report"
 	"github.com/aquasecurity/trivy/pkg/types"
+	funk "github.com/thoas/go-funk"
 )
 
+func PrDiffResults(r report.Results) (reports report.Results, err error) {
+	for _, v := range r {
+		// is head file and not exist in base
+		fileInBase := false
+		if strings.Contains(v.Target, "head") {
+			toBase := strings.ReplaceAll(v.Target, "head", "base")
+			for _, vBase := range r {
+				if vBase.Target == toBase {
+					fileInBase = true
+				}
+			}
+			// this is new file take full report
+			if !fileInBase {
```

Comment:
Extract to a new function that checks if file exists in base(return true if it exists)